routes/repl/_UserMenu.html

```diff
@@ -0,0 +1,58 @@
+<div class='user' on:mouseenter='set({ showMenu: true })' on:mouseleave='set({ showMenu: false })'>
+	<span>{{$user.displayName}}</span>
```

Comment:
It's possible to have no displayName (I don't have one!) and so this should probably fall back to the username. Right now I'm seeing 'null' here.

Reply:
huh! fixed